dipy/sims/voxel.py

```diff
@@ -15,6 +15,17 @@
 #
 diffusion_evals = np.array([1500e-6, 400e-6, 400e-6])
 
+# DT and KT for a voxel of well aligned fibers 
+# parameters are based on the simulations from:
+#
+#    Neto Henriques et al., "Exploring the 3D geometriy of the diffusion 
+#    kurtosis tensor - Impact on the development of robust tractography 
+#    procedures and novel biomarkers", NeuroImage, 2015; 111:85-99.
+#
+diffusion_tensor = np.array([1638e-6, 444e-6, 444e-6, 0, 0, 0])
```

Comment:
Can we instead change the evals? The parameters diffusion_tensor and kurtosis_tensor are the defaults for a voxel of well aligned fibers and assuming the multi-compartmental model's parameters on Table 1 of Neto Henriques et al. (2015), so I want to keep then consistent to each other. Another option is to remove these global variables at all, and require users always giving the DT and KT as input of function single_diffkurt_tensors.